packages/link/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@meshconnect/web-link-sdk",
-  "version": "3.0.2",
+  "version": "3.1.0",
```

Comment:
A Patch number update should be enough, since no new functionality was added to the SDK. https://semver.org/spec/v2.0.0.html

Reply:
Thanks! I read the doc and have since updated the versions appropriately.